yarn.lock

```diff
@@ -1602,166 +1594,166 @@
-  integrity sha512-sdxA6gVlE9TkkajyV8VeFj6m6ut8yC1RIqK72TedFYDiFQ+R1NyOepVpda6O9vQ3nwDUkf+phl1YPBCAiI3Viw==
+"@endo/import-bundle@^1.4.0":
+  version "1.4.0"
+  resolved "https://registry.yarnpkg.com/@endo/import-bundle/-/import-bundle-1.4.0.tgz#7a93013833367fcbb4e783e23114b88c966b2d99"
+  integrity sha512-sW3Gf1Icmy9LMSnDS+DDEnmqB0wBpq0njrW3ollb/pX5ExK03A5dLhZ4tqhIvPkRxXaycfIv7pbnH5aM2qQMuQ==
   dependencies:
     "@endo/base64" "^1.0.9"
-    "@endo/compartment-mapper" "^1.5.0"
-    "@endo/errors" "^1.2.9"
+    "@endo/compartment-mapper" "^1.6.0"
+    "@endo/errors" "^1.2.10"
     "@endo/where" "^1.0.9"
-    ses "^1.11.0"
+    ses "^1.12.0"
 
-"@endo/init@^1.1.8":
-  version "1.1.8"
-  resolved "https://registry.yarnpkg.com/@endo/init/-/init-1.1.8.tgz#a6c1e8d10b1098de3d566ddd55c7b33797519ed9"
-  integrity sha512-YSbLRjjT0qv/oemzuEJTCjZKj7zFnWgBrA2Kt33RwQuJq2p24oxcUTzEzyui4Xs9yBWaG+hnWV79r1qs3cpR5w==
+"@endo/init@^1.1.9":
+  version "1.1.9"
+  resolved "https://registry.yarnpkg.com/@endo/init/-/init-1.1.9.tgz#e2506abdb17bcd38ab6685608b75cc7789899e61"
+  integrity sha512-YDl4zHUv54ghuau2tSC08xwSDu/QhmTmOdxWgJNlj29bwe5n3s3G8j5jbzx6FNqdZq0W2ow9uzKdpEczpFy+Uw==
   dependencies:
     "@endo/base64" "^1.0.9"
-    "@endo/eventual-send" "^1.3.0"
-    "@endo/lockdown" "^1.0.14"
-    "@endo/promise-kit" "^1.1.9"
-
-"@endo/lockdown@^1.0.14":
-  version "1.0.14"
-  resolved "https://registry.yarnpkg.com/@endo/lockdown/-/lockdown-1.0.14.tgz#8562b97e4e7da542949b711717aa5f2bbf2c930e"
-  integrity sha512-V4buKRPDY6IpeaE9jc0Gmj1bZwOLCEJdzvFoqwZA23vdX9YXhACBPc8C5/lK00HrsfOxPOCo4YmsBY8LN9RfBQ==
+    "@endo/eventual-send" "^1.3.1"
+    "@endo/lockdown" "^1.0.15"
+    "@endo/promise-kit" "^1.1.10"
+
+"@endo/lockdown@^1.0.15":
+  version "1.0.15"
+  resolved "https://registry.yarnpkg.com/@endo/lockdown/-/lockdown-1.0.15.tgz#35030381a34ac7d68031517142b7bca2b261f3d9"
+  integrity sha512-UFnsGkT7n4J4jtkFbgo8f4ENGctcbZqtIvGMWGLEO2p7ZsvCnOZhStuHZKt9Vx1u/5IAv/X8/+RCUkyGTsM03w==
+  dependencies:
+    ses "^1.12.0"
+
+"@endo/marshal@^1.6.4":
+  version "1.6.4"
+  resolved "https://registry.yarnpkg.com/@endo/marshal/-/marshal-1.6.4.tgz#47f76871edf1745f171d0ca7c0c4408b7b79aabf"
+  integrity sha512-HpTNDDPLjLAHmE5WjWxObmkqpJAyDX4yDARfOzqjEGvJTtdj8hePbV3wfrTMqg2DdaUOuKN2xFAjSCsN6S+GXA==
+  dependencies:
+    "@endo/common" "^1.2.10"
+    "@endo/errors" "^1.2.10"
+    "@endo/eventual-send" "^1.3.1"
+    "@endo/nat" "^5.1.0"
+    "@endo/pass-style" "^1.5.0"
+    "@endo/promise-kit" "^1.1.10"
+
+"@endo/module-source@^1.3.0":
+  version "1.3.0"
+  resolved "https://registry.yarnpkg.com/@endo/module-source/-/module-source-1.3.0.tgz#633aa6d23091c10190fefb63ebd934dd17260914"
+  integrity sha512-waFivuJYvPRKP+W01XAw+Hz1X81Bb05IDhrcFJXFaVil7B6hxByQ6aUeYeGFugCfke51DzvzSE1RbDnWy2u/7Q==
   dependencies:
-    ses "^1.11.0"
+    "@babel/generator" "^7.26.3"
+    "@babel/parser" "~7.26.2"
+    "@babel/traverse" "~7.25.9"
+    "@babel/types" "~7.26.0"
```

Comment:
Any reason module-source is using `~` specifiers here ? It seems to cause some duplicate babel packages in agoric-sdk.

Nothing actionable in this release sync, but to consider relaxing in the next release.

Reply:
Yes, Babel’s listing experimental_preserveFormat as a flag they will want to remove in a minor bump.